src/Bridge/Doctrine/Orm/Extension/PaginationExtension.php

```diff
@@ -267,6 +269,19 @@ private function getPaginationParameter(Request $request, string $parameterName,
         return $request->query->get($parameterName, $default);
     }
 
+    private function addCountToContext(QueryBuilder $queryBuilder, array $context): array
+    {
+        if (!($context['graphql'] ?? false)) {
+            return $context;
+        }
+
+        if (isset($context['filters']['last']) && !isset($context['filters']['before'])) {
+            $context['count'] = (new DoctrineOrmPaginator($queryBuilder))->count();
```

Comment:
Should I assume that partial pagination is not compatible with Graphql?

Reply:
Yes. We could make it compatible later.

Reply:
The code you annotated is only if you are doing backwards pagination though.